activejob/test/cases/queue_adapter_test.rb

```diff
@@ -49,6 +49,7 @@ class QueueAdapterTest < ActiveJob::TestCase
     child_job_three = Class.new(ActiveJob::Base)
 
     assert_not_nil child_job_three.queue_adapter
```

Comment:
```suggestion
```
We could replace the nil check entirely.

Reply:
Done 🚀